htdocs/admin/modules.php

```diff
@@ -685,7 +684,7 @@
         	}
         	else
         	{
-        		print '<a class="reposition" href="modules.php?id='.$objMod->numero.'&amp;module_position='.$module_position.'&amp;action=reset&amp;value=' . $modName . '&amp;mode=' . $mode . $param . '">';
+        		print '<a name="'.$modName.'" href="modules.php?id='.$objMod->numero.'&amp;module_position='.$module_position.'&amp;action=reset&amp;value=' . $modName . '&amp;mode=' . $mode . $param . '">';
```

Comment:
Mmmm it was like an "experiment", because there was no need to use JS if there is a bundled-in feature in the browser.

But it is OK if you don't find it nice enough.